Content.Client/SimpleStation14/Examine/CharacterInformation/Systems/CharacterInformationSystem.cs

```diff
@@ -0,0 +1,149 @@
+    [Dependency] private readonly IPrototypeManager _prototype = default!;
+    [Dependency] private readonly IConfigurationManager _config = default!;
+
+    private CharacterInformationWindow? _window;
+
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        _window = new CharacterInformationWindow();
+
+        SubscribeLocalEvent<CharacterInformationComponent, GetVerbsEvent<ExamineVerb>>(OnGetExamineVerbs);
+    }
+
+
+    private void OnGetExamineVerbs(EntityUid uid, CharacterInformationComponent component, GetVerbsEvent<ExamineVerb> args)
+    {
+        var verb = new ExamineVerb()
+        {
+            Act = () =>
+            {
+                // TODO: Better name?
+                Do(args.Target);
+            },
+            Text = Loc.GetString("character-information-verb-text"),
+            Message = Loc.GetString("character-information-verb-message"),
+            Category = VerbCategory.Examine,
+            Disabled = !_examine.IsInDetailsRange(args.User, uid),
+            Icon = new SpriteSpecifier.Texture(new ResourcePath("/Textures/Interface/VerbIcons/sentient.svg.192dpi.png")),
+            ClientExclusive = true,
+        };
+
+        args.Verbs.Add(verb);
+    }
+
+
+    private void Do(EntityUid uid)
+    {
+        if (_window == null)
+            return;
+
+        string? name = null;
+        string? job = null;
+        string? flavorText = null;
+
+        // Get ID from inventory, get name and job from ID
+        if (_inventory.TryGetSlotEntity(uid, "id", out var idUid))
+        {
+            var id = GetId(idUid);
+            if (id is not null)
+            {
+                var info = GetNameAndJob(id);
+
+                name = info.Item1;
+                job = info.Item2;
+            }
+        }
+
+        // Fancy job title
+        if (!string.IsNullOrEmpty(job))
+        {
+            var test = job.Replace(" ", "");
+            // Command will be last in the list
+            // TODO: Make this not revolve around this fact ^
+            var departments = _prototype.EnumeratePrototypes<DepartmentPrototype>().OrderBy(d => d.ID).Reverse();
+            var department = departments.FirstOrDefault(d => d.Roles.Contains(test));
+
+            if (department is not null)
+            {
+                // Department (ex: Command or Security)
+                var dept = string.Join(" ", Loc.GetString($"department-{department.ID}").Split(' ').Select(s => s[0].ToString().ToUpper() + s[1..].ToLower()));
+                // Redo the job title with the department color and department (ex: Captain (Command) or Security Officer (Security))
+                job = $"[color={department.Color.ToHex()}]{job} ({dept})[/color]";
+            }
+        }
+
+        // Get and set flavor text
+        if (!_config.GetCVar(CCVars.FlavorText))
+        {
+            flavorText = Loc.GetString("character-information-ui-flavor-text-disabled");
+        }
```

Comment:
I don't love how off putting this window is for servers without flavour text enabled, but I get that "solving" that through any means is likely far more effort than it's worth.